packages/itwinui-css/src/bridge.scss

```diff
@@ -312,7 +312,6 @@
     border-radius: calc(4px + 6px);
```

Comment:
I'm wondering about the other overrides in here. Should we also adjust the focus outline offset for the regular borderless tabs?

Currently (without theme bridge), it is difficult to distinguish a "focused + selected" tab from "focused + non-selected" tab.

Reply:
That looks ok, let's go with that?

Reply:
Additional test image in https://github.com/iTwin/iTwinUI/pull/2541/commits/e15565fe286607340057f01d89bb5b023d70608e

Reply:
Forgot to mention: `outline-offset: -6px` can now be removed from L310 above.

Reply:
Done in https://github.com/iTwin/iTwinUI/pull/2541/commits/226099a892264214f1ee1d1863e5e05a5a1daa6e